examples/index.js

```diff
@@ -10,21 +13,27 @@ class CodeEditor extends React.Component {
       code: '// type your code... \n',
     }
   }
-  editorDidMount(editor) {
+
+  onChange = (newValue, e) => {
+    console.log('onChange', newValue, e); // eslint-disable-line no-console
```

Comment:
I think we can remove these log statements (or at least comment them out) in a follow up PR. I appreciate that you didn't change semantics in this refactor PR.